unstable/lens/hash-pluck.scrbl

```diff
@@ -0,0 +1,16 @@
+#lang scribble/manual
+
+@(require lens/doc-util/main)
+
+@title{Slicing a hash table}
```

Comment:
Perhaps "Plucking from a hash table"? It'll probably go in a general "hash" section eventually anyway, so it's not that big deal